Sources/Protocols/ConfigurableView.swift

```diff
@@ -23,17 +23,17 @@
 import Foundation
 
 /**
- *  protocol which ensures that specific type can create view model and can apply new view state with view model
+ *  Protocol that ensures that specific type can create view model and can apply new view state with view model
  */
-public protocol AbstractViewModelProtocol {
+public protocol ConfigurableView {
     associatedtype ViewModelType
```

Comment:
Associated type should be renamed. Same about description for method.

Reply:
renamed to what? any suggestions?

Reply:
Think about smth called `Configurator` instead of `ViewModelType` according to `ConfigurableView`.